.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+# Motivation
+<!-- Describe the reason for this change. What problem does it solve or what feature does it add? -->
+
+# Changes
+<!-- List the changes made in this pull request. Be concise but specific. -->
+
+# Testing
```

Comment:
I've added a **Documentation** section as well when I've made changes to these templates in other SDKs. Since our SDKs are publicly used, most changes have a DX impact.

Reply:
usually just a link to linear ticket for updating the docs, or a link to a PR where the docs have been updated (or are wip) or n/a. Just something to demonstrate that updating the docs is tracked.